pkg/pgmodel/migrations/sql/idempotent/base.sql

```diff
@@ -462,6 +462,21 @@ BEGIN
 END;
 $$;
 
+CREATE OR REPLACE FUNCTION SCHEMA_CATALOG.delete_series(
```

Comment:
Let's change the name to reflect that this deletes on the row for the series not all the info from the data tables as well.
```suggestion
CREATE OR REPLACE FUNCTION SCHEMA_CATALOG.delete_series_catalog_row(
```

Reply:
Definitely!